src/pages/api/[...slug].astro

```diff
@@ -3,7 +3,7 @@ import { getCollection } from '../../utils/getCollection';
 import Prose from '../../components/Prose.vue';
 import BaseLayout from '../../layouts/BaseLayout.astro';
 import customComponents from '../../utils/customComponents';
-import H2 from '../../components/H2.astro';
+import ApiH2 from '../../components/ApiH2.astro';
```

Comment:
Yea we could use the rule of 3

Reply:
I think the correct way to do this would be to use a plugin to transform all H2s, perhaps [rehype-autolink-headings](https://github.com/rehypejs/rehype-autolink-headings). I agree though @AndyClifford because we have that visible anchor tag and the styling is different it isn't so easy. I think given the time we have left this is ok for now.